draft-ietf-anima-brski-prm.md

```diff
@@ -338,7 +338,7 @@ In constrained environments it may be provided based on COSE {{RFC9052}} and {{R
 
 An abstract overview of the BRSKI-PRM protocol can be found on slide 8 of {{BRSKI-PRM-abstract}}.
 
-To support mutual trust establishment between the domain registrar and pledges not directly connected to the customer site/domain, this document specifies the exchange of authenticated self-contained objects (the voucher-request/response as known from BRSKI and the enrollment-request/response as introduced by BRSKI-PRM) with the help of a registrar-agent.
+To support mutual trust establishment between the domain registrar and pledges not directly connected to the customer site/domain, this document specifies the exchange of authenticated self-contained objects (the voucher-request/response as known from BRSKI and the enrollment-request/response as introduced by BRSKI-AE) with the help of a registrar-agent.
```

Comment:
@stfries It seems weird for this document to refer to itself this way.  I'm unclear if the enrollment/response is really in this document or in AE.  Could this reference be wrong?  If not, then should we always refer to self as "This Document"?)

Reply:
That is weird indeed and actually a double reference. I dont even think it is necessary to have these specifics in the introduction. Proposal:

"To support mutual trust establishment between the domain registrar and pledges not directly connected to the customer site/domain, this document specifies the exchange of authenticated self-contained objects with the help of a registrar-agent. "

Reply:
edited and rebased.